test/e2e/serial/workload_placement_test.go

```diff
@@ -259,6 +266,154 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, schedulerName)
 		})
+
+		When("a RuntimeClass exist in the cluster", func() {
+			var rtClass *nodev1.RuntimeClass
```

Comment:
Being RuntimeClass a non-namespaced resource I have created the context to use beforeEach and AfterEach to properly delete the object and avoid cluster polution.

Reply:
Makes sense, I'm still not too sold about using `BeforeEach`/`AfterEach` when we have just a single `It` - instead of having, say, a simple `defer`. But that's not a blocking issue.